charts/kubescape-operator/templates/kubescape/servicemonitor.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
   labels:
     app: {{ .Values.kubescape.name }}
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    {{- with .Values.kubescape.serviceMonitor.customLabels }}
```

Comment:
I think we should have "labels" and not "customLabels".
What do you think?

Reply:
Yes, quite.